ZeachCobbler.user.js

```diff
@@ -5065,11 +5092,15 @@ $('#AcidMode-checkbox').change(function(){
     if(!!this.checked){
         $('#AcidMode-textbox').removeAttr("disabled");
     } else {
-        $('#AcidMode-textbox').attr({disabled:"disabled"})
+        $('#AcidMode-textbox').attr({disabled:"disabled"});
     }
     cobbler.MenuAcidMode = !!this.checked;
 });
-if(cobbler.MenuAcidMode){$('#AcidMode-checkbox').prop('checked', true);}else{ $('#AcidMode-textbox').attr({disabled:"disabled"})}
```

Comment:
Q: Why you here deleting the line and "expanding it" (sorry, my English is very high level)?